dp-identity-data-migration/user_extraction.go

```diff
@@ -0,0 +1,208 @@
+package main
+
+import (
+	"encoding/csv"
+	"fmt"
+	"strings"
+
+	// "io/ioutil"
+	"os"
+
+	"time"
+
+	"github.com/ONSdigital/dp-zebedee-sdk-go/zebedee"
+	"github.com/google/uuid"
+)
+
+type cognito_user struct {
+	username,
+	name,
+	given_name,
+	family_name,
+	middle_name,
+	nickname,
+	preferred_username,
+	profile,
+	picture,
+	website,
+	email,
+	email_verified,
+	gender,
+	birthdate,
+	zoneinfo,
+	locale,
+	phone_number,
+	phone_number_verified,
+	address,
+	updated_at,
+	mfa_enabled string
+}
+
+func populate_Header() (header cognito_user) {
+	header = cognito_user{
+		username:              "cognito:username",
+		name:                  "name",
+		given_name:            "given_name",
+		family_name:           "family_name",
+		middle_name:           "middle_name",
+		nickname:              "nickname",
+		preferred_username:    "preferred_username",
+		profile:               "profile",
+		picture:               "picture",
+		website:               "website",
+		email:                 "email",
+		email_verified:        "email_verified",
+		gender:                "gender",
+		birthdate:             "birthdate",
+		zoneinfo:              "zoneinfo",
+		locale:                "locale",
+		phone_number:          "phone_number",
+		phone_number_verified: "phone_number_verified",
+		address:               "address",
+		updated_at:            "updated_at",
+		mfa_enabled:           "cognito:mfa_enabled",
+	}
+	return header
+}
+func convert_cognito_user_to_slice(input cognito_user) (output []string) {
```

Comment:
No need to include the variable name here, or brackets, this will do (also, for data structures, I prefer passing by reference):

`func convert_cognito_user_to_slice(input cognito_user) *[]string {`